cli/context/context.go

```diff
@@ -114,6 +114,8 @@ type PackCtx struct {
 
 	PreInstallScriptFile  string
 	PostInstallScriptFile string
+
+	SystemUnitParams common.PackSystemUnitParams
```

Comment:
I think that context should store only file name, it is filled on parsing `pack` flags. But processing this file is pack-specific, so it can be performed only there and this structure should exist only in `pack`. Moreover, `PackSystemUnitParams` structure should be defined in `pack/systemd_dir`.